### ISSUE TITLE
Error running `ls` or `auto-renew`: not calculating time before renewal

### ISSUE THREAD

**Issue by jnm** [dokku/dokku-letsencrypt#83]
Hello, I'm receiving the following message while trying to run `dokku letsencrypt:ls` or `dokku letsencrypt:auto-renew`:

```
/var/lib/dokku/plugins/available/letsencrypt/functions: line 33: [: : integer expression expected
```

Complete output:

```
$ dokku letsencrypt:ls
-----> App name           Certificate Expiry        Time before expiry        Time before renewal
/var/lib/dokku/plugins/available/letsencrypt/functions: line 33: [: : integer expression expected
data.redacteddomain.org       2016-12-14 00:22:00       68d, 4h, 46m, 24s

$ dokku letsencrypt:auto-renew
=====> Auto-renewing all apps...
/var/lib/dokku/plugins/available/letsencrypt/functions: line 33: [: : integer expression expected
       data.redacteddomain.org still has  days left before renewal
=====> Finished auto-renewal
```

I'm running dokku 0.7.1 and dokku-letsencrypt 0.8.6:

```
$ dokku version
0.7.1
$ dokku plugin | grep letsencrypt
  letsencrypt          0.8.6 enabled    Automated installation of let's encrypt TLS certificates
```

Here's the output of `ls` with `dokku trace on`:
http://0bin.net/paste/qJlLyfQne4iYHrvS#cLhqkM-sN8bygKNEAKRQfbRmlRaTGScjwwcRx6d4zNi
...and the output of `auto-renew`:
http://0bin.net/paste/krVCH9wN5oJM+VZL#UhCV6BJ92Wq0CaMexf3QiqrYs+G0tPrVrYpj8nucbI+

**Comment by sseemayer** [comment]
Thanks for your detailed report! It seems that when querying for the value of the DOKKU_LETSENCRYPT_GRACEPERIOD config variable, an empty string is returned, leading to problems in renewal time calculation.

What's the output of `dokku config --global` and `dokku config <yourapp>` (I only need the value of the DOKKU_LETSENCRYPT_GRACEPERIOD setting in case one is set)?

**Comment by ebeigarts** [comment]
The problem is detailed in [dokku#2447](https://github.com/dokku/dokku/issues/2447) and will be fixed in [dokku#2454](https://github.com/dokku/dokku/pull/2454). In the meantime you can set a dummy global environment variable (`dokku config:set --global DUMMY=1`) to avoid this issue.

**Comment by jnm** [comment]
Thanks, @ebeigarts! That workaround fixes this problem for me.

Sorry for the delay, @sseemayer. I don't actually have `DOKKU_LETSENCRYPT_GRACEPERIOD` set anywhere.
